### PR TITLE
Authorizer: Improve granter/revoker interface

### PR DESCRIPTION
**Pull request by facuspagnuolo**
### MERGE AFTER #1224

### CODE REVIEW

**Review by nventuro — APPROVED** [review]
_Nice_, it's amazing how much this simple improvement makes things more readable and easier to understand!

**Comment by facuspagnuolo** [comment]
Glad you liked it @nventuro! I have just pushed another minor enhacement

File: pkg/vault/contracts/authorizer/TimelockAuthorizer.sol [hunk 1]
```diff
@@ -343,12 +351,8 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication {
...
         require(block.timestamp >= scheduledExecution.executableAt, "ACTION_NOT_EXECUTABLE");
         if (scheduledExecution.protected) {
-            bool isAllowed = hasPermissionOrWhatever(
-                EXECUTE_ACTION_ID,
-                msg.sender,
-                address(this),
-                bytes32(scheduledExecutionId)
-            );
+            bytes32 executeScheduledActionId = getActionId(EXECUTE_ACTION_ID, bytes32(scheduledExecutionId));
+            bool isAllowed = hasPermission(executeScheduledActionId, msg.sender, address(this));
```
Comment (nventuro) [inline pkg/vault/contracts/authorizer/TimelockAuthorizer.sol:355]: This is not the same as the old version, is it? With this change it is not possible to have global execution rights

Comment (nventuro) [reply pkg/vault/contracts/authorizer/TimelockAuthorizer.sol:355]: (I don't think we needed those, just pointing that out)

**Pull request merged**